public/scripts/worktrollyUI.js

```diff
@@ -164,11 +164,8 @@ $("#submitCreateNewTask").click(function() {
     var createNewTaskFunction = firebase.functions().httpsCallable('createNewTask');
     createNewTaskFunction({ Title: title, Description: des, Priority: priority, Difficulty: difficulty, Creator: creator, Assignee: assignee, EstimatedTime: estimatedTime, Status: status, Category: category, SprintNumber: sprintNumber, StoryPointNumber: storyPointNumber }).then(result => {
         console.log(result.data);
-        newPage = "dashboard";
-        uiLoader();
+        location.reload();
     });
-    $('input').val('');
-    $('select').val('');
 });
```

Comment:
undo this change.
And add .onSnapshot instead of .get()

`use this format to read`
db.collection("cities").doc("SF")
    .onSnapshot(function(doc) {
     ........
    });

do these changes everywhere in dashboard.js

Reply:
and also remove the catch method after reading the data.

Reply:
Done the appropriate changes